src/lib/storage/create_iterable_from_column.ipp

```diff
@@ -0,0 +1,12 @@
+#pragma once
```

Comment:
What is an ipp file?

Reply:
It’s an **i**mplementation file where the implementation of template classes and methods goes into. In other projects I used to use `.hpp` with this type of file and `.h` for header files. Boost uses `.ipp` so I did as well. The explanation why we need to separate implementation from declaration in this case is in a comment in the `.hpp` file.